app/docs/0.1.1beta-2/cli.md

```diff
@@ -0,0 +1,132 @@
+---
+title: CLI Reference
+---
+
+# CLI Reference
+
+Kong comes with a ***Command Line Interface*** (refered to as "CLI") which lets you perform operations such as starting and stopping a node. Each command is run in the context of a single node, since Kong as no cluster awareness yet.
+
+Almost every CLI command needs access to your configuration file in order to be aware of what is your node's NGiNX working directory, (known as the "prefix path" for those familiar with NGiNX), and referenced as `nginx_working_dir` in your Kong configuration file.
```

Comment:
I've seen other places in the website where it is "NGiNX", so I was only trying to be consistent. This needs to be updated everywhere if still present.